.changeset/legal-ghosts-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: `messageArgs` to `declaration-property-value-keyword-no-deprecated`
```

Comment:
note

#8595 didn't actually add `messageArgs` to that rule.

Reply:
Ya, for some reason, I missed this rule...